### PR TITLE
Use industrial_ci & docker

### PR DESCRIPTION
**Pull request by artivis**
See #45 .

### CODE REVIEW

**Comment by artivis** [comment]
@130s can you approve PR to master as well ?
@cbandera Hi, sorry for pinging.

File: .travis.yml [hunk 1]
```diff
@@ -1,122 +1,46 @@
...
+    - ROS_DISTRO="jade"     ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+    - ROS_DISTRO="jade"     ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - ROS_DISTRO="jade"     PRERELEASE=false
+    - ROS_DISTRO="kinetic"  ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+    - ROS_DISTRO="kinetic"  ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - ROS_DISTRO="kinetic"  PRERELEASE=true
+    - ROS_DISTRO="lunar"    ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+    - ROS_DISTRO="lunar"    ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - ROS_DISTRO="lunar"    PRERELEASE=true
+    #- ROS_DISTRO="melodic"  ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
```
Comment (130s) [inline .travis.yml:31]: ROS Melodic is supported by industrial_ci, if that's what you wanted :)

**Review by 130s — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (artivis) [reply .travis.yml:31]: Last time I tried I got some error message stating it wasn't thus the comment. I'll enable it in a next release ;)  
Thanks for the review.

**Comment by artivis** [comment]
So I still can't merge : `At least 1 approving review is required by reviewers with write access` :'(
@cbandera gentle ping as it seems you are the only one whom can approves this.

**Review by cbandera — APPROVED** [review]

**Pull request merged**